basic.py

```diff
@@ -103,11 +104,11 @@ def sub_event_catcher(event):
         last_events[event.meta['host']] = event
 
 
-@match(regex=r'^s([^ a-zA-Z0-9])(.*?)\1(.*?)(?:\1(.+?))?$')
+@match(regex=SED_REGEX)
 def sub_match(event, sep, pattern, replacement, flags, *args, **kwargs):
     """Matches any substitute string (as sed would), applies this substitution to
-    the user's last message and sents it back to processing.
-    """
+    the user's last message and sends it back to processing."""
```

Comment:
@pepol yeah, but you have omitted the second part

``` This way, Emacs' fill-paragraph command can be used on it. ```

If that's the reason for that I am very much OK with leaving it this way here.